src/envo/envo-edit.owl

```diff
@@ -40841,7 +40841,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501164> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501165> (oncology unit facility)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501165> "A hospital unit facility which is used to provide oncology."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501165> "A hospital unit facility which is used to provide oncology."@en)
```

Comment:
@ddooley This definition doesn't match the pattern of the other hospital units: it ends with "... provide oncology". I think you could be more descriptive here. E.g., "A hospital unit facility which provides treatment for cancer".

Reply:
Suggestion accepted!